src/fsharp/FSharp.Core/control.fs

```diff
@@ -729,7 +729,7 @@ namespace Microsoft.FSharp.Control
                     )
                     
         let startThreadWithTrampoline (trampolineHolder:TrampolineHolder) (f : unit -> FakeUnitValue) =
-#if FX_NO_THREADPOOL
+#if FX_THREADPOOL
                 if not (ThreadPool.QueueUserWorkItem((waitCallbackForQueueWorkItemWithTrampoline trampolineHolder), f |> box)) then
```

Comment:
Looks like `FX_NO_THREADPOOL` should just be dropped in favor of `FX_NO_THREAD` (which is already defined, in all the same places as `FX_NO_THREADPOOL`), and the definition of the fake guys should also be conditional on `FX_NO_THREAD`.  That would keep everything unified I think.